src/verify_cert.rs

```diff
@@ -788,30 +798,24 @@ mod tests {
     }
 
     #[cfg(feature = "alloc")]
-    fn verify_chain(
-        trust_anchor: &CertificateDer<'_>,
-        intermediates_der: &[Vec<u8>],
-        ee_cert: &CertificateDer<'_>,
+    fn verify_chain<'a>(
```

Comment:
Because `build_chain_inner()` now yields a type that captures a bunch of lifetimes from the input here, I had to restructure the test utilities to make the lifetimes work out.